debian/rules

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+# hardening=+all fails, so at least we should push the minimum
+export DEB_BUILD_MAINT_OPTIONS=hardening=+bindnow+relro+pie
+
+%:
+	dh $@ --buildsystem=waf
+
+override_dh_auto_install:
+	dh_auto_install $@
+	mv debian/tmp/etc/ndn/client.conf.sample debian/tmp/etc/ndn/client.conf
```

Comment:
Oh I see what you mean - I didn't realize that this is Endless-specific. That's fine then.